File: ClientApp/src/app/models/comment.ts
import { User } from "./user";

export class Comment {
    id: number;
    body: string;
    timestamp: string;
    authorId: number;
    postId: number;
    authorPseudo: string;

    constructor(data: any) {
        if (data) {
            this.id = data.id;
            this.body = data.body;
            this.timestamp = data.timestamp;
            this.authorId = data.authorId;
            this.postId = data.postId;
            this.authorPseudo = '';
        }
    }
}

File: prid1920-g10/Models/Comment.cs
using System;
using System.Diagnostics;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using Microsoft.EntityFrameworkCore;
using System.Text.RegularExpressions;
using System.ComponentModel.DataAnnotations.Schema;
using System.Linq;

namespace prid1920_g10.Models {
    public class Comment {
        [Key]
        public int Id { get; set; }
        public string Body { get; set; }
        public DateTime Timestamp { get; set; }
        public int AuthorId { get; set; }
        public int PostId { get; set; }
        
        public virtual Post Post { get; set; }
        public virtual User User { get; set; }
    }
}